error.go

```diff
@@ -38,10 +39,21 @@ func (e *errResponse) Response() *http.Response {
 	return e.response
 }
 
-// NewError ...
-func NewError(expectedRespCode []int, resp *http.Response) Error {
+// NewResponseError ...
+func NewResponseError(expectedRespCode []int, resp *http.Response) Error {
 	return &errResponse{
 		expectedResponseCode: expectedRespCode,
 		response:             resp,
 	}
 }
+
+// ScreeningErrors ...
+type ScreeningErrors []error
+
+func (s *ScreeningErrors) Error() string {
+	buf := bytes.Buffer{}
+	for _, err := range *s {
+		buf.WriteString(err.Error())
```

Comment:
@chandradeepak you might wanna put a space or new line here.

Reply:
@chandradeepak nevermind, I pushed a commit fixing this. Once the build is done, I'll merge this.